### ISSUE TITLE
Disable and hide next/previous buttons

### ISSUE THREAD

**Issue by pburtchaell** [ardaogulcan/react-native-keyboard-accessory#5]
## Background

Nice component!

I'm using this component on a form, but I don't want the next/previous buttons—just the done button.

I've disabled the next/previous buttons like so: 

```js
<KeyboardAccessoryNavigation
  nextDisabled
  previousDisabled
  nextButton={(<View />)}
  previousButton={(<View />)}
/>
```

## Request

It would be nice if I could disable and hide the next/previous buttons like so:

```js
<KeyboardAccessoryNavigation
  nextHidden
  previousHidden
/>
```

This would not render the next/previous buttons at all.

**Comment by pburtchaell** [comment]
If you think this is a reasonable request, I am happy to make a PR with the feature. 😄

**Comment by ardaogulcan** [comment]
Hi @pburtchaell, It makes sense and I'll be glad to have your PR or I can implement it next week. Thank you for your interest.

**Comment by ardaogulcan** [comment]
As the PR merged I'm closing this issue. This feature is available in version >= [0.0.7](https://github.com/ardaogulcan/react-native-keyboard-accessory/releases/tag/0.0.7)
